### DISCUSSION TITLE
Static Files caching (and clearing)

### DISCUSSION THREAD

**Discussion by lunchboxer** [beginner-corp/begin-community#78] [Q&A] [answered]
I have a static file folder with my repo that I've recently changed completely. Unfortunately, none of this is reflected in the deployment. I assume this is intended behavior, just cacheing, but I wonder how I can bust the cache and/or get it to rebuild the cache based on the actual contents of the bucket. The problem isn't on the client side, as I get the same stale files with curl. Any hints?

**Comment by lunchboxer** [comment, marked as answer]
After a few hours, I suspected this was not the intended caching behavior and looked at my config again. I had deleted the static folder and expected it to disappear from the server, but this apparently doesn't happen quickly. Updating an existing, configured static folder works as expected. I solved the issue by configuring a new blank folder in app.arc.

**Reply by ryanblock** [reply]
Ah, nice find. Yes, we do purge unused static files when you configure `@static prune true` in your arc file.
